models/nft/ethereum/nft_ethereum_floor_price.sql

```diff
@@ -0,0 +1,126 @@
+{{ config(
+    alias = 'floor_price',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'nft_contract_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "sector",
+                                "nft",
+                                \'["Henrystats"]\') }}'
+    )
+}}
+
+WITH 
+
+src_data as (
+    SELECT 
+        *,
+        date_trunc('day', block_time) as block_date
+    FROM 
+    {{ ref('nft_trades') }} 
+    WHERE blockchain = 'ethereum'
+    AND trade_type = 'Single Item Trade'
```

Comment:
nft.trades is already setup so each row contains only one `token_id` traded.
It might thus be possible to drop this conditional.

One row might still contain a trade with multiple ERC1155s though, if you don't want to deal with that I suggest using `number_of_items = 1` as filter.

Reply:
okaiee ser, I'll update thank you very much!

Reply:
updated